packages/coordinator/src/store/appointments/actions/CoordinatorUpdateAction.ts

```diff
@@ -0,0 +1,38 @@
+import { CoordinatorUpdate } from "@zm-blood-components/common";
+import { ThunkAction } from "../../store";
```

Comment:
What is `ThunkAction` ?

Reply:
Redux Thunk is a middleware for redux that lets you dispatch other actions and access the app state when running an action: https://github.com/reduxjs/redux-thunk

What I import here is the typing of thunk actions for our app (i.e. thunk + the structure of our state with appointments and stuff)